proxysql/tests/common.py

```diff
@@ -171,3 +173,21 @@
         'query_rules_metrics',
     ],
 }
+
+
+def mock_executor(result=()):
+    def executor(_):
+        return result
+
+    return executor
+
+
+def create_query_manager(*args, **kwargs):
+    executor = kwargs.pop('executor', None)
+    if executor is None:
```

Comment:
I think you're right that it's not really necessary since I hardcode the executor to be `mock_executor` in this instance, but I think it can be useful for future tests that need to create a custom query manager for tests.

Reply:
Sure, that's fine!